config/features.yml

```diff
@@ -199,6 +199,9 @@ features:
   gibct_school_ratings:
     actor_type: user
     description: Show/Hide the school ratings section of the Comparison Tool School Profile Page.
+  hlr_v2:
+    actor_type: user
+    descriiption: Enable showing Higher-Level Review v2
```

Comment:
typo - descriiption should be description

Reply:
oops, thanks